src/Features/Navigation/Menu.php

```diff
@@ -275,13 +279,17 @@ public static function add_plugin_item( $args ) {
 	 *      'url'        => (string) URL or callback to be used. Required.
 	 *      'migrate'    => (bool) Whether or not to hide the item in the wp admin menu.
 	 *      'menuId'     => (string) The ID of the menu to add the category to.
+	 *      'order'      => (int) Menu item order.
 	 *    ).
 	 */
 	public static function add_plugin_category( $args ) {
+		if ( ! isset( $args['parent'] ) ) {
+			unset( $args['order'] );
+		}
+
```

Comment:
If you can supply an `order` here now, do we have logic in place to make sure the top level ones in the "Extensions" section are still alphabetical? That was a requirement, so just want to make sure thats not overwritten.

Reply:
>Make sure top-level items cannot be re-ordered (should remain alphabetical).

Oh, I should read before commenting 😨